### DISCUSSION TITLE
Nimib Speaking Hours - February - Sun, Feb 25, 9:00 AM (CET)

### DISCUSSION THREAD

**Discussion by HugoGranstrom** [pietroppeter/nimib#232] [Nimib speaking hours]
The next Nimib Speaking Hours will take place Sunday the 25th of February at 9:00 AM (CET).
Here is a link to add it to your calendar: https://calndr.link/event/uVocooZNC6

**Comment by pietroppeter** [comment]
notes for today:
- meta agenda:
  - informal chat 
  - agenda
  - check notes from last meeting
  - what has happened since last time
  - did we have some pending answering/feedback tasks for users/contributors
  - what's the very non strict plan for next "spreeeeent"
  - (at least one diversion or it is not official)
  - be mindful of time
  - schedule next call
- agenda
  - check notes from last meeting
    - ok, no pending tasks from last time
  - what has happened since last time
    - Hugo made very nice exploration on serializing with jsony OOP stuff
    - Pietro idea of submitting and Literate Programming and Explorable Explanations talk to PyData confs (with NimibLand as 1/3 of topic)
    - Pietro: Nim for Pythonistas accepted at PyCon Italy (will be streamed)
    - Pietro FOSDEM was cool (NimConf in Tromso?) 
  - did we have some pending answering/feedback tasks for users/contributors
    - dlesnoff on code line numbers [x]
    - neroist on tree block (nimibex) [x]
  - to discuss:
    - big refactoring
      - https://github.com/nimib-land/nblog/issues/20 
    - nimib.land?
      - let's focus on the other stuff first 
  - what's the very non strict plan for next "spreeeeent"
    - Hugo
      - blogpost https://github.com/nimib-land/nblog/issues/20
    - Pietro
      - complete nblog migration with redirect pages
      - maybe also do the nimib migration
      - working on refactoring
  - (at least one diversion or it is not official)
  - be mindful of time
  - moar discussion
    - we could record the 15' we actually spend discussing nimib?
      - we could try it next time
      - it means we might have a youtube channel
  - schedule next call
    - Sat March 23rd 10:30 
- diversions:
  - https://thefloatingcontinent.com/blog/llm-bullshit/
  - https://twitter.com/pietroppeter/status/1761311087373246549
  - https://www.youtube.com/watch?v=Rg3r3gG4nQo&t=645s

**Reply by pietroppeter** [reply]
I was sure I was forgetting a diversion. This talk from Fosdem by the maintainer of `brew` is really good (especially the necessity of establish boundaries): https://fosdem.org/2024/schedule/event/fosdem-2024-2029-open-source-in-2024-boundaries-burnout-business/
